backend/.coveragerc

```diff
@@ -0,0 +1,4 @@
+[report]
```

Comment:
could you also add a banner to the README for coverage?

Reply:
I can add it but it will be updated after we merge to master